main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitrise-io/go-steputils/v2/export"
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/errorutil"
+	. "github.com/bitrise-io/go-utils/v2/exitcode"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-io/go-xcode/v2/destination"
+	"github.com/bitrise-io/go-xcode/v2/xcodeversion"
+	"github.com/bitrise-steplib/bitrise-step-xcode-test-shard-calculation/step"
+)
+
+func main() {
+	exitCode := run()
+	os.Exit(int(exitCode))
+}
+
+func run() ExitCode {
+	logger := log.NewLogger()
+	testSharder := createStep(logger)
+
+	config, err := testSharder.ProcessConfig()
+	if err != nil {
+		logger.Println()
+		logger.Errorf(errorutil.FormattedError(fmt.Errorf("Failed to process Step inputs: %w", err)))
```

Comment:
You may get "MISSING" errors in the logs if there are format string-like characters in any logs.
```suggestion
		logger.Errorf("%s", errorutil.FormattedError(fmt.Errorf("Failed to process Step inputs: %w", err)))
```